src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -606,6 +606,14 @@ protected function requestFromRequest(Request $request, $changeHistory = true)
         return $this->request($request->getMethod(), $request->getUri(), $request->getParameters(), $request->getFiles(), $request->getServer(), $request->getContent(), $changeHistory);
     }
 
+    /**
+     * Update the server setting from the given URI.
```

Comment:
`Updates`

Reply:
Oh, that's a private method, we don't document private method except when really needed. So, this docblock and the next should be removed.

Reply:
I didn't notice! Sorry. I reverted.